spec/features/merge_pact_spec.rb

```diff
@@ -0,0 +1,57 @@
+describe "Merging a pact" do
+
+  let(:pact_content) { load_fixture('a_consumer-a_provider.json') }
+  let(:path) { "/pacts/provider/A%20Provider/consumer/A%20Consumer/version/1.2.3" }
+  let(:response_body_json) { JSON.parse(subject.body) }
+
+  subject do
+    patch path, pact_content, {'CONTENT_TYPE' => 'application/json' }; last_response
+  end
+
+  context "when a pact for this consumer version does not exist" do
+    it "returns a 201 Created" do
+      expect(subject.status).to be 201
+    end
+
```

Comment:
For the use case I'm looking at, pushing from parallel builds, creating a new resource through `PATCH` makes the most sense - otherwise I'd need to do something funky with retrying as a PUT request if the resource doesn't exist yet, and that introduces problems with concurrency and dropping pieces of the pact.